advanced_source/cpp_export.rst

```diff
@@ -0,0 +1,364 @@
+`save <https://pytorch.org/docs/master/jit.html#torch.jit.ScriptModule.save>`_
+on the module and pass it a filename::
+
+  traced_script_module.save("model.pt")
+
+This will produce a ``model.pt`` file in your working directory. We have now
+officially left the realm of Python and are ready to cross over to the sphere
+of C++.
+
+Step 3: Loading Your Script Module in C++
+------------------------------------------
+
+To load your serialized PyTorch model in C++, your application must depend on
+the PyTorch C++ API -- also known as *LibTorch*. The LibTorch distribution
+encompasses a collection of shared libraries, header files and CMake build
+configuration files. While CMake is not a requirement for depending on
+LibTorch, it is the recommended approach and will be well supported into the
+future. For this tutorial, we will be building a minimal C++ application using
+CMake and LibTorch that simply loads and executes a serialized PyTorch model.
+
+A Minimal C++ Application
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Let's begin by discussing the code to load a module. The following will already
+do:
+
+.. code-block:: cpp
+
+  #include <torch/op.h> // One-stop header.
+
+  #include <iostream>
+  #include <memory>
+
+  int main(int argc, const char* argv[]) {
+    if (argc != 2) {
+      std::cerr << "usage: example-app <path-to-exported-script-module>\n";
+      return -1;
+    }
+
+    // Deserialize the ScriptModule from a file using torch::jit::load().
+    std::shared_ptr<torch::jit::script::Module> module = torch::jit::load(argv[1]);
+
+    assert(module != nullptr);
+    std::cout << "ok\n";
+  }
+
+The ``<torch/op.h>`` header encompasses all relevant includes from the LibTorch
+library necessary to run the example. Our application accepts the file path to
+a serialized PyTorch ``ScriptModule`` as its only command line argument and
+then proceeds to deserialize the module using the ``torch::jit::load()``
+function, which takes this file path as input. In return we receive a shared
+pointer to a ``torch::jit::script::Module``, the equivalent to a
+``torch.jit.ScriptModule`` in C++. For now, we only verify that this pointer is
+not null. We will examine how to execute it in a moment.
+
+Depending on LibTorch and Building the Application
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Assume we stored the above code into a file called ``example-app.cpp``. A
+minimal ``CMakeLists.txt`` to build it could look as simple as:
+
+.. code-block:: cmake
+
+  cmake_minimum_required(VERSION 3.0 FATAL_ERROR)
+  project(custom_ops)
+
+  find_package(Torch REQUIRED)
+
+  add_executable(example-app example-app.cpp)
+  target_include_directories(example-app PRIVATE "${TORCH_INCLUDE_DIRS}")
+  target_link_libraries(example-app "${TORCH_LIBRARIES}")
+  target_compile_definitions(example-app PRIVATE -D_GLIBCXX_USE_CXX11_ABI=0)
+  set_property(TARGET example-app PROPERTY CXX_STANDARD 11)
+
+The last thing we need to build the example application is the LibTorch
```

Comment:
Add a warning that the C++ version and the conda version you exported with need to be matched. In the future we will provide a better BC story. Not in preview though.